app/_src/gateway/kong-enterprise/aws-iam-auth-to-rds-database.md

```diff
@@ -36,6 +36,9 @@ Before you enable the AWS IAM authentication, you must configure your AWS RDS da
    
    {:.warning}
    > **Warning:** You **can't** change the value of the environment variables you used to provide the AWS credential after booting {{site.base_gateway}}. Any changes are ignored.
+{% if_version gte:3.8.x %}
+   - {{site.base_gateway}} also supports role assuming which allows you to use a different IAM role to authenticate to the database. If you want to use role assuming, make sure the original IAM role Kong uses has the correct permission to do role assuming to the target IAM role, and the target IAM role has the correct permission to connect to the database using IAM authentication.
```

Comment:
```suggestion
   - If you want to assume a role, make sure the original IAM role that Kong uses has the correct permission to assume the role of the target IAM role, and the target IAM role has the correct permission to connect to the database using IAM authentication.
```
You don't need the first sentence, this entry even being here implies that Kong Gateway supports it.

Rephrased to use more common language around AssumeRole. "Role assuming" sounds like it could be right, but AWS doesn't use that phrasing in their docs.

Reply:
> Rephrased to use more common language around AssumeRole. "Role assuming" sounds like it could be right, but AWS doesn't use that phrasing in their docs.
Good point! I'll be careful with that next time